src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js

```diff
@@ -849,7 +849,12 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
               <BarChart
                 data={chartData}
                 height={CHART_HEIGHT}
-                margin={{ top: 20, right: 50, bottom: 50, left: 55 }}
+                margin={{
+ top: 20,
+right: 50,
+bottom: 50,
+left: 55 
+}}
```

Comment:
what's up with the indent here?

Reply:
very good question, I tried to copy-paste the original file from cdave branch to correct/revert the changes to this file but it didn't work.

need to fix my vscode next week

Reply:
fixed in a merge conflict in another PR!

Reply:
thanks! also fixed it in mine.